packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.tsx

```diff
@@ -380,9 +403,89 @@ export class ArtistDashboardPage extends Component<
     )
   }
 
+  renderUSDCSection() {
```

Comment:
For the sake of moving this page away from class components, it probably makes sense to put the USDC stuff into a sub component and pass it props.